packages/webdriverio/src/commands/browser/execute.js

```diff
@@ -24,6 +24,21 @@
     });
  * </example>
  *
+ * <example>
+    :execute.js
+    // Cucumber context - Step in feature file
+    When I do 1 + 2
+    // Implementation of the step in js file
+    When(/^I do \d + \d', function (x, y) {
+        var result = browser.execute(function(a, b) {
+            // browser context - you may not access client or console (alternatively you can use alert() to see the output, but not recommended)
+            return a + b;
+        }, x, y);
+        // node.js context - client and console are available
+        console.log(result.value); // outputs: 3
+    });
+ * </example>
```

Comment:
I don't see a reason why we should add the same example in a different framework. The command is framework agnostic so it works the same way whereever you run it. Happy to remove the `it("...")` part of the example if this is confusing.

Reply:
I understand your point @christian-bromann. The key highlight here how to pass the arguments we have got from cucumber step to the browser.execute(). This is where I am kind of stuck for a couple of hours before getting to know the usage. The only intention is that it would help a few like that...
Inspired by @klamping comment https://stackoverflow.com/a/46304389